### PR TITLE
Prod sync

### PR DESCRIPTION
**Pull request by just-a-node**
## Type of change

- [ ] Docs change / dependency upgrade
- [ ] Configuration / tooling changes
- [ ] Refactoring
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires changes in customer code

## Related Issue(s)

Fixes

### CODE REVIEW

**Review by rhlsthrm — DISMISSED** [review]

File: packages/agents/sequencer/src/config.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ import { version } from "../package.json";
 import { SequencerConfig, SequencerConfigSchema } from "./lib/entities";
 
 const DEFAULT_AUCTION_WAIT_TIME = 10_000;
-const DEFAULT_EXECUTION_WAIT_TIME = 120_000;
```
Comment (preethamr) [inline packages/agents/sequencer/src/config.ts:13]: We want to keep this change.

**Review by preethamr — CHANGES_REQUESTED** [review]
Looks like `prod` needs to be merged into `prod-sync` ?

**Review by preethamr — CHANGES_REQUESTED** [review]
Looks like `prod` needs to be merged into `prod-sync` ?

**Comment by just-a-node** [comment]
> Looks like `prod` needs to be merged into `prod-sync` ?

Are we missing things? This was branched from `prod`, merging `testnet-prod` into it.

**Pull request closed without merging**